app/views/kaui/refunds/_form.html.erb

```diff
@@ -158,8 +179,32 @@
         }
         recomputeRefundAmountAndValidateAmount();
         validateInvoiceItemAmount(id);
+        checkSelectAllCheckboxStatus();
     };
 
+    /*
+    * When clicking select all checkbox - select each items, disable amount and recompute total refund amount
+    */
+    var onClickInvoiceItemsSelectAll = function(event){
+        var isChecked = false;
+        if ($(this).is(':checked')) {
+            isChecked = true;
+        }
+        $('input').filter(function() { return this.id.match(/tf_adj_/) }).each(function() {
+            var textFieldId = this.id;
+            var checkboxId = textToCheckboxId(this.id);
+            if(checkboxId && textFieldId){
```

Comment:
Checking textFieldId might be redundant as there is a check of `textToCheckboxId(textFieldId)`

Reply:
`textFieldId ` is removed